docs/how-to-guides/validator-setup.mdx

```diff
@@ -9,93 +9,96 @@ We will also set up a broadcaster service, TSS service, and EVM identity saver s
 ## Node (rarimo-core)
 
 To run the node, follow these steps:
+
 1. Download the following:
-  [genesis file](https://storage.cloud.google.com/rarimo-mainnet/core/genesis.json),
-  [address book](https://storage.googleapis.com/rarimo-mainnet/core/addrbook.json),
-  [APP file](https://storage.googleapis.com/rarimo-mainnet/core/app.toml) and
-  [rarimo core binary file](https://storage.googleapis.com/rarimo-mainnet/core/rarimo-core)
+   [genesis file](https://storage.cloud.google.com/rarimo-mainnet/core/genesis.json),
+   [address book](https://storage.googleapis.com/rarimo-mainnet/core/addrbook.json),
+   [APP file](https://storage.googleapis.com/rarimo-mainnet/core/app.toml) and
+   [rarimo core binary file](https://storage.googleapis.com/rarimo-mainnet/core/rarimo-core)
 
 1. Generate validator configs using `rarimo-core` binary
 
 1. Connect to `34.66.205.183`.
-This IP address is used for submitting TXs and p2p connections.
+   This IP address is used for submitting TXs and p2p connections.
 
 1. Connect to the RPC node: `39072210913bb52a6444543ad160b04668d58210@34.66.205.183:26656`
 
 1. Execute script to set environments:
 
-  ```bash
```

Comment:
The linter broke markdown. Please re-check the nested lists

<img width="762" alt="image" src="https://github.com/rarimo/docs/assets/11010444/754aeb89-3d01-4a06-87f5-d25a039b8a7a">